scripts/install_submitter.py

```
@@ -0,0 +1,768 @@
+                    except (OSError, IOError) as e:
+                        logger.warning("Failed to install plugin to %s: %s", version_name, e)
+                else:
+                    logger.warning(
+                        "Site-packages not found for %s: %s", version_name, site_packages
+                    )
+            else:
+                logger.debug("VRED %s not found at: %s", version_name, vred_path)
+
+        if success_count > 0:
+            logger.info("Plugin installed to %d VRED version(s)", success_count)
+            return True
+
+        logger.warning("Plugin not installed to any VRED versions")
+        return False
+
+    def _get_vred_installation_paths(self) -> dict[str, Optional[Path]]:
+        """Get VRED installation paths for different versions."""
+        autodesk_dir = Path("C:/Program Files/Autodesk")
+        vred_paths = {}
+
+        if autodesk_dir.exists():
+            # Find VRED Pro 2025 (VREDPro-17.*) - select highest version
+            vred_17_paths = [path for path in autodesk_dir.glob("VREDPro-17*") if path.is_dir()]
+            if vred_17_paths:
+                vred_paths["VRED Pro 2025"] = max(vred_17_paths, key=self._extract_version_number)
+
+            # Find VRED Pro 2026 (VREDPro-18.*) - select highest version
+            vred_18_paths = [path for path in autodesk_dir.glob("VREDPro-18*") if path.is_dir()]
+            if vred_18_paths:
+                vred_paths["VRED Pro 2026"] = max(vred_18_paths, key=self._extract_version_number)
+
+        return vred_paths
+
+    def _extract_version_number(self, path: Path) -> tuple[int, ...]:
+        """Extract version number from VRED path as a tuble (major, minor)"""
+        # Extract version from "VREDPro-17.3" -> (17, 3)
+        match = re.search(r"VREDPro-(\d+)\.(\d+)", path.name)
+        if match:
+            return tuple(int(x) for x in match.groups())
+        return (0, 0)  # Fallback for invalid format
```

This _might_ be a place where we'd want to raise an exception over passing through a fallback

---

Right, I have fixed this to raise `ValueError` for invalid version formats instead of returning (0, 0)